src/main/features/claim/form/models/eligibility/validDefendant.ts

```diff
@@ -0,0 +1,29 @@
+export class ValidDefendant {
```

Comment:
I do not think class name correctly reflects the purpose of that class. Options more reflect who is making a claim not who is the defendant on the claim e.g. `REPRESENTATIVE_CLAIM`. Shall we name the class `ClaimType` or something that indicates this is about the claimant not defendant?

Reply:
After reading this i messaged you in slack and talked about it. To summarise the conversation, im changing the name to ClaimType as you suggested